source/Orders.h

```diff
@@ -78,6 +78,7 @@ class Orders {
 	bool HasHarvest() const;
 
 	bool IsEmpty() const;
+	size_t count() const;
```

Comment:
Should this be capitalized? We already use `IsEmpty` instead of `empty`.
```suggestion
	size_t Count() const;
```

Reply:
If we consider this class to be an extension of `std::bitset`, then I think it should be lower-case.
`std::bitset` has no `empty` method.

Reply:
It does have `any`, though.